File: exercicio3.php
<?

define("TAM_X", 3);
define("TAM_Y", 2);

function read_matrix()
{
  $matriz = array();

  for ($i = 0; $i < TAM_X; $i++) {
    $matriz[$i] = array();

    for ($j = 0; $j < TAM_Y; $j++) {
      echo "Digite o número da posição $i,$j: ";

      $matriz[$i][$j] = intval(fgets(STDIN));
    }
  }

  return $matriz;
}

function calc_matrizes($matriz_a, $matriz_b)
{
  $soma_matriz =  array();
  
  for ($i = 0; $i < count($matriz_a); $i++) {
    $soma_matriz[$i] =  array();

    for ($j = 0; $j < count($matriz_a[$i]); $j++) {
      $soma_matriz[$i][$j] = $matriz_a[$i][$j] + $matriz_b[$i][$j];
    }
  }

  return $soma_matriz;
}

function imprime_matriz($matriz, $titul_matriz) {
  echo "$titul_matriz\n";
  
  for ($i = 0; $i < count($matriz); $i++) {
    echo implode(", ", $matriz[$i])."\n";
  }

  echo "\n";
}

$matriz_a = read_matrix();
$matriz_b = read_matrix();
$matriz_c = calc_matrizes($matriz_a, $matriz_b);

imprime_matriz($matriz_a, "Matriz A");
imprime_matriz($matriz_b, "Matriz B");
imprime_matriz($matriz_c, "Matriz C");

File: exercicio3.py
matrizA = []
matrizB = []

def alimentaMatriz(matriz):
    for linha in range(3):
        matriz.append([])
        for coluna in range(2):
            print("Digite a linha " + str(linha + 1) + " e coluna " + str(coluna + 1) + ":")
            matriz[linha].append(int(input("-> ")))
    print()

def imprimeMatriz(matriz, texto):
    print(texto)
    for i in matriz:
        print(i)
    print()

def somaMatriz(matriz1, matriz2):
    matrizResultado = []
    for linha in range(3):
        matrizResultado.append([])
        for coluna in range(2):
            matrizResultado[linha].append(matriz1[linha][coluna] + matriz2[linha][coluna])
    return matrizResultado

print("Matriz A (3x2)")
alimentaMatriz(matrizA)
print("Matriz B (3x2)")
alimentaMatriz(matrizB)
matrizC = somaMatriz(matrizA, matrizB)
imprimeMatriz(matrizA, "Matriz A")
imprimeMatriz(matrizB, "Matriz B")
imprimeMatriz(matrizC, "Matriz C")